src/jit/flowgraph.cpp

```diff
@@ -4857,7 +4859,8 @@ void Compiler::fgFindJumpTargets(const BYTE* codeAddr, IL_OFFSET codeSize, BYTE*
                     fgObserveInlineConstants(opcode, pushedStack, isInlining);
                 }
                 break;
-
+            case CEE_RET:
+                retBlocks++;
```

Comment:
Note sure, need to look more at what `fgMakeBasicBlocks` is doing. Maybe we can actually ignore the `.tail` prefix, the IL code should contain a `ret` anyway for the method to be valid.